cosmosdb/gremlin/create.sh

```diff
@@ -0,0 +1,69 @@
+!/bin/bash
```

Comment:
can the file names be more specific something like - cosmosdb/gremlin/create-database, cosmosdb/gremlin/update-throughput...

Reply:
It's not just creating a database though, it is creating a database and a graph, same with the others too. Also, because I've been advised to include multiple operations into a single script I prefer to keep the script name more topical than fully descriptive. Additionally, this also ensures the script names can be read in GitHub.